### ISSUE TITLE
The twisted version of raw sockets doesn't handshake the max length correctly

### ISSUE THREAD

**Issue by fhriley** [crossbario/crossbar#1207]
The twisted version of raw sockets sets the max length in the handshake reply to 16777216, but the router won't actually let you send any message larger than 131072.

**Comment by oberstet** [comment]
Yep, because the maximum message size being accepted by Crossbar.io currently isn't configurable: https://github.com/crossbario/crossbar/blob/master/crossbar/router/protocol.py#L581

So we need to make it configurable;)

**Comment by fhriley** [comment]
Allowing configuration of the length doesn't fix the bug I'm reporting. The bug is that during handshake, the router's handshake reply always says the maximum length is 16777216. If the max message size is 131072, as it is currently set to, the router's handshake reply should say the maximum length is 131072. As is, the implementation is not implementing the spec correctly.

**Comment by oberstet** [comment]
Yes, right. Fixing the issue here will involve fixing this issue https://github.com/crossbario/autobahn-python/issues/922, resulting in spec conformance.
